### ISSUE TITLE
Duplicated title in the response of bundled requests

### ISSUE THREAD

**Issue by pmadruga** [spencermountain/wtf_wikipedia#218]
Doing a bundle request is returning a repeated response (in some parts). For example:

```
 wtf
        .fetch(['apple', 'pizza'], 'en', {
          'Api-User-Agent': 'obfuscated@gmail.com'
        })
        .then(docList => {
          console.warn(docList)
        });
```

returns 

<img width="447" alt="screenshot 2018-12-05 at 22 48 35" src="https://user-images.githubusercontent.com/3617636/49546189-f5cc4300-f8df-11e8-9a7b-4d370ec9fd79.png">

It's possible to see that the title is the same but the data is different. 

Thanks in advance!

**Comment by spencermountain** [comment]
thanks! i saw that a few days back and thought my eyes were bad.
will check it out.

**Comment by spencermountain** [comment]
fixed in `7.1.0`
thanks .
